lite/proxy/proxy.go

```diff
@@ -38,9 +38,11 @@ func StartProxy(c rpcclient.Client, listenAddr string, logger log.Logger, maxOpe
 	core.SetLogger(logger)
 	mux.HandleFunc(wsEndpoint, wm.WebsocketHandler)
 
-	_, err = rpc.StartHTTPServer(listenAddr, mux, logger, rpc.Config{MaxOpenConnections: maxOpenConnections})
-
-	return err
+	l, err := rpc.Listen(listenAddr, rpc.Config{MaxOpenConnections: maxOpenConnections})
```

Comment:
can we rename `rpc` to `rpcserver` for consistency

Reply:
done, thanks